plugins/modules/synchronize.py

```diff
@@ -601,6 +614,9 @@ def main():
 
     cmd.append(shlex_quote(source))
     cmd.append(shlex_quote(dest))
+    if quiet:
+        cmd.append('--quiet')
+
```

Comment:
This break the change detection as @felixfontein was mentioning [here](https://github.com/ansible-collections/ansible.posix/pull/220#issuecomment-2155447386)

Plus we probably want to keep the changes in `stdout_lines`. It's only `msg` that we want to suppress them from.